demisto_sdk/commands/lint/linter.py

```diff
@@ -508,6 +518,12 @@ def _linter_to_commands(self):
             ),
         }
 
+    def _get_files_to_push(self):
+        files_to_push = [(self._pack_abs_dir, "/devwork")]
+        if CAN_MOUNT_FILES:
+            files_to_push.append((CONTENT_PATH, "/content"))  # type: ignore[arg-type]
```

Comment:
we can avoid this `type: ignore` by adding type hints to files_to_push

Reply:
The problems is that `self._pack_abs_dir` is `wcmpath.pathlib`` and `CONTENT_PATH` is a normal `pathlib`.
We can use Union but we need to import the regular `pathlib` for it.